operator/validator/controller.go

```diff
@@ -379,8 +379,10 @@ func (c *controller) handleWorkerMessages(msg network.DecodedSSVMessage) error {
 			SyncCommRoots:     c.syncCommRoots,
 			DomainCache:       c.domainCache,
 		}
+		msgID := ssvMsg.GetID()
+		identifier := msgID[:]
 		ncv = &committeeObserver{
-			CommitteeObserver: validator.NewCommitteeObserver(committeeObserverOptions),
+			CommitteeObserver: validator.NewCommitteeObserver(identifier, committeeObserverOptions),
```

Comment:
if we store it as `MessageID` inside the committee observer (see my other comment), we can just pass `ssvMsg.GetID()`